src/operator/tensor/elemwise_binary_op_basic.cc

```diff
@@ -10,6 +10,16 @@ namespace mxnet {
 namespace op {
 MXNET_OPERATOR_REGISTER_BINARY(elemwise_add)
 .add_alias("_add").add_alias("_plus").add_alias("_Plus")
+.describe(R"code(Adds arguments element-wise.
+
+Example::
+
+  >>> x = mx.nd.array([1, 2, 3, 4])
```

Comment:
@mli @piiswrong : I don't see this operator defined in the Python files. Any idea how to keep the description in the cc file and add the example elsewhere specific to Python?